x-pack/plugins/cloud_security_posture/public/pages/rules/rules_counters.tsx

```diff
@@ -0,0 +1,319 @@
+import { useCspBenchmarkIntegrationsV2 } from '../benchmarks/use_csp_benchmark_integrations';
+import { DASHBOARD_COUNTER_CARDS } from '../compliance_dashboard/test_subjects';
+import { CspCounterCard } from '../../components/csp_counter_card';
+import { useKibana } from '../../common/hooks/use_kibana';
+
+const EvaluationPieChart = ({ failed, passed }: { failed: number; passed: number }) => {
+  const {
+    services: { charts },
+  } = useKibana();
+
+  return (
+    <Chart size={{ height: 30, width: 30 }}>
+      <Settings
+        theme={[
+          {
+            partition: {
+              linkLabel: { maximumSection: Infinity, maxCount: 0 },
+              outerSizeRatio: 0.75,
+              emptySizeRatio: 0.7,
+            },
+          },
+        ]}
+        baseTheme={charts.theme.useChartsBaseTheme()}
+      />
+      <Partition
+        id={'evaluation-pie-chart'}
+        data={[
+          {
+            label: i18n.translate('xpack.csp.rulesPage.evaluationPieChart.failedTitle', {
+              defaultMessage: 'Failed',
+            }),
+            value: failed,
+          },
+          {
+            label: i18n.translate('xpack.csp.rulesPage.evaluationPieChart.passedTitle', {
+              defaultMessage: 'Passed',
+            }),
+            value: passed,
+          },
+        ]}
+        valueGetter="percent"
+        valueAccessor={(d) => d.value}
+        layout={PartitionLayout.sunburst}
+        layers={[
+          {
+            groupByRollup: (d: { label: string }) => d.label,
+            shape: {
+              fillColor: (label) =>
+                label.toLowerCase() === RULE_PASSED.toLowerCase()
+                  ? statusColors.passed
+                  : statusColors.failed,
+            },
+          },
+        ]}
+      />
+    </Chart>
+  );
+};
+
+export const RulesCounters = () => {
+  const { http } = useKibana().services;
+  const rulesPageParams = useParams<{ benchmarkId: string; benchmarkVersion: string }>();
+  const getBenchmarks = useCspBenchmarkIntegrationsV2();
+  const navToFindings = useNavigateFindings();
+  const cspmIntegrationLink = useCspIntegrationLink(CSPM_POLICY_TEMPLATE) || '';
+  const kspmIntegrationLink = useCspIntegrationLink(KSPM_POLICY_TEMPLATE) || '';
+
+  const benchmarkRulesStats = getBenchmarks.data?.items.find(
+    (benchmark) =>
+      benchmark.id === rulesPageParams.benchmarkId &&
+      benchmark.version === rulesPageParams.benchmarkVersion
+  );
+
+  if (!benchmarkRulesStats) {
+    return <></>;
+  }
+
+  const benchmarkDynamicValues: Record<
+    BenchmarksCisId,
+    {
+      integrationType: string;
+      integrationName: string;
+      resourceName: string;
+      integrationLink: string;
+      learnMoreLink: string;
+    }
+  > = {
+    cis_aws: {
+      integrationType: 'CSPM',
+      integrationName: 'AWS',
+      resourceName: 'Accounts',
+      integrationLink: cspmIntegrationLink,
+      learnMoreLink: 'https://ela.st/cspm-get-started',
```

Comment:
I think URL values as the `learnMoreLink, should ideally come from a constants file or a configuration, to ease updates and reusability.

Reply:
yeah right now we have a mess with external links. i have a separated task to do concentrate all our links under one object